config/routes.rb

```diff
@@ -27,6 +27,7 @@
   get "/cart", :to => "spree/orders#edit", :as => :cart
   put "/cart", :to => "spree/orders#update", :as => :update_cart
   put "/cart/empty", :to => 'spree/orders#empty', :as => :empty_cart
+  match '/orders/:id/token/:token' => 'spree/orders#show', :via => :get, :as => :token_order
```

Comment:
if it's only via `:get` we better replace `match` with `get` as in line 27. If I'm not mistaken it is something that gets deprecated in Rails 4 and will need fixing. I remember dealing with `match` routes in the upgrade I did.

Reply:
yeah, this is done [here](https://github.com/spree/spree/commit/c73dbbc49e57a17d38d1332e3a1ea2eb9af15667#diff-77e25f6625b6d4d28d71cdbe70021df8) in spree.

I added a commit to do it.